examples/9_Ahneman_Science_CN_Coupling/example_Ahenman.ipynb

```diff
@@ -369,26 +369,19 @@
     "\n",
     "# Analyses: UPLC\n",
     "# Note using LCMS because UPLC is not an option\n",
-    "outcome.analyses['UPLC analysis'].type = reaction_pb2.ReactionAnalysis.LCMS\n",
+    "outcome.analyses['UPLC analysis'].type = reaction_pb2.Analysis.LCMS\n",
     "outcome.analyses['UPLC analysis'].details = ('UPLC using 3 µL of 0.0025 M di-tert-butylbiphenyl solution in DMSO external standard')\n",
     "outcome.analyses['UPLC analysis'].instrument_manufacturer = 'Waters Acquity'\n",
     "\n",
     "# Define product identity\n",
     "prod_2a = outcome.products.add() \n",
-    "prod_2a.compound.identifiers.add().CopyFrom(\n",
-    "    reaction_pb2.CompoundIdentifier(value=r'FC(C1=CC=C(NC2=CC=C(C)C=C2)C=C1)(F)F', type='SMILES')\n",
-    ")\n",
+    "prod_2a.identifiers.add(value=r'FC(C1=CC=C(NC2=CC=C(C)C=C2)C=C1)(F)F', type='SMILES')\n",
     "prod_2a.is_desired_product = True\n",
     "\n",
-    "# Define product yield from results table\n",
-    "# Yields are reported with insignificant digits\n",
-    "prod_2a.compound_yield.CopyFrom(\n",
-    "    reaction_pb2.Percentage(value=10.65781182) \n",
-    ")\n",
-    "\n",
     "# The UPLC analysis was used to confirm both identity and yield\n",
-    "prod_2a.analysis_identity.append('UPLC analysis')\n",
-    "prod_2a.analysis_yield.append('UPLC analysis')\n",
+    "prod_2a.measurements.add(type='IDENTITY', analysis_key='UPLC analysis')\n",
+    "prod_2a.measurements.add(type='YIELD', analysis_key='UPLC analysis', percentage=dict(value=10.65781182),\n",
+    "                             uses_internal_standard=True)\n",
```

Comment:
Again, are these calibrated?

Reply:
That's not completely clear to me. Each distinct product _was_ synthesized separately and used to identify retention times at least. I can't see whether yields were calculated based on a true calibration curve though. I would think they are given that all reported yields are properly between 0 and 100...